backend/src/interfaces/github.interface.ts

```diff
@@ -0,0 +1,29 @@
+import { Repository } from '@octokit/webhooks-types';
+import { GithubRepo } from '../constants';
```

Comment:
Can we use absolute imports? I don't remember if we have that setup yet.

Reply:
Apparently yes, that is setup :slightly_smiling_face:

Reply:
I changed it and pushed the change so I have no idea :laughing: